Meta/tweet-commits.js

```diff
@@ -13,11 +16,14 @@ const T = new Twit({
     const githubEvent = JSON.parse(fs.readFileSync(0).toString());
     const tweets = [];
     for (const commit of githubEvent["commits"]) {
-        tweets.push(
-            `${commit["message"].substring(0, 240)}\nAuthor: ${commit["author"]["name"]}\n${
-                commit["url"]
-            }`
-        );
+        const authorLine = `Author: ${commit["author"]["name"]}`;
+        const maxMessageLength = tweetLength - authorLine.length - twitterLinkLength - 2; // -2 for newlines
+        const commitMessage =
+            commit["message"].length > maxMessageLength
+                ? commit["message"].substring(0, maxMessageLength - 2) + "…" // Ellipsis counts as 2 characters
+                : commit["message"];
+
+        tweets.push(`${commitMessage}\n${authorLine}\n${commit["url"]}`);
```

Comment:
I know this is just taken from the existing script, but `foo.bar.baz` is way more JavaScript-y than `foo["bar"]["baz"]`.